app/controllers/actions/CheckOtherCountryRegistrationFilter.scala

```diff
@@ -55,13 +60,18 @@ class CheckOtherCountryRegistrationFilterImpl @Inject()(
         if isActiveNotInAmend(activeMatch) =>
         Some(Redirect(controllers.filters.routes.AlreadyRegisteredOtherCountryController.onPageLoad(activeMatch.memberState)))
 
-      case Some(activeMatch)
-        if isQuarantinedNotInAmend(activeMatch) =>
-        Some(Redirect(
-          controllers.filters.routes.OtherCountryExcludedAndQuarantinedController.onPageLoad(
-            activeMatch.memberState,
-            getEffectiveDate(activeMatch))
-        ))
+      case Some(activeMatch) if isQuarantinedNotInAmend(activeMatch) =>
+        val effectiveDate = getEffectiveDate(activeMatch)
+        val quarantineCutOffDate = LocalDate.now.minusYears(2)
```

Comment:
when using `.now` you should pass `clock` which you can inject into the controller, this allows us to control when "now" is via clock inject in specs